xapian-letor/feature/feature.cc

```diff
@@ -36,10 +39,17 @@ void
 Feature::operator=(const Feature & o)
 {
     LOGCALL_VOID(API, "Feature::operator=", o);
-    internal = o.internal;
+    feature_db = o.feature_db;
+    feature_query = o.feature_query;
+    feature_doc = o.feature_doc;
+    termfreq = o.termfreq;
+    inverse_doc_freq = o.inverse_doc_freq;
+    doc_length = o.doc_length;
+    collection_length = o.collection_length;
+    collection_termfreq = o.collection_termfreq;
 }
```

Comment:
They are not copyable or assignable. The assignment operator and copy constructor are private. In the initial definition of the functions, internal was assigned as `internal = o.internal` for both the functions that is why I have assigned each variable to have a proper function definition. Although both methods are private and to be never called. Should I remove the assignment of each member variable in both the functions and just let there be log calls  ??